### ISSUE TITLE
Bug: Clicking on the ends of hold notes with hold note or hold mid option selected doesn't change ease type.

### ISSUE THREAD

**Issue by MasterCoder21** [crash5band/MikuMikuWorld#44]
Version: MMW v3.0.0
Platform: Windows 11
How to re-create:
1. Place a hold note in any chart.
2. Select the Hold or Hold Mid option in the toolbar.
3. Click on the ends of the previously placed hold note or on another hold mid.

In previous versions, clicking on a hold note end or hold mid changed easing, but it doesn't seem to work on v3.0.0.  I understand one can change easing from the Options panel, but simply clicking on a hold note or hold mid is very convenient.

**Comment by crash5band** [comment]
Yes I'm aware. I just hadn't enough time to fully test it with the new notes. I'll bring it back very soon™️

**Comment by crash5band** [comment]
Functionality restored with 484cb5b
